oc-cluster

```diff
@@ -61,6 +73,13 @@ OS_DEFAULT_PROJECT="myproject"
 #
 __RESTART=0
 
+function which-ip {
+  if [[ "$__PLATFORM" == "linux" ]]; then
+    /sbin/ifconfig $1 | grep "inet " | awk -F: '{print $1}' | awk '{print $2}'
```

Comment:
ahh I see why ... I think below you call this without an argument, therefore it gets all ips